content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -137,11 +137,11 @@ It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0
 The format also includes extended `vixie cron` step values. As explained in the [FreeBSD manual](https://www.freebsd.org/cgi/man.cgi?crontab%285%29):
 
 > Step values can be	used in	conjunction with ranges.  Following a range
-> with ``/<number>''	specifies skips	of the number's	value through the
-> range.  For example, ``0-23/2'' can be used in the	hours field to specify
+> with ``/<number>`` specifies skips	of the number's	value through the
+> range.  For example, ``0-23/2`` can be used in the	hours field to specify
 > command execution every other hour	(the alternative in the	V7 standard is
-> ``0,2,4,6,8,10,12,14,16,18,20,22'').  Steps are also permitted after an
-> asterisk, so if you want to say ``every two hours'', just use ``*/2''.
+> ``0,2,4,6,8,10,12,14,16,18,20,22``).  Steps are also permitted after an
+> asterisk, so if you want to say ``every two hours``, just use ``*/2``.
 
```

Comment:
Hello @ebriand. It appears that this paragraph is a direct quote from, https://www.freebsd.org/cgi/man.cgi?crontab(5), including extra spaces and punctuation. Possibly clean-up the extra spaces? Thanks.